gitlab/resource_gitlab_group_ldap_link.go

```diff
@@ -20,6 +20,9 @@ func resourceGitlabGroupLdapLink() *schema.Resource {
 		Create: resourceGitlabGroupLdapLinkCreate,
 		Read:   resourceGitlabGroupLdapLinkRead,
 		Delete: resourceGitlabGroupLdapLinkDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceGitlabGroupLdapLinkImporter,
```

Comment:
It would be nice to use the `schema.ImportStatePassthroughContext` importer instead of writing our custom one.

To make use of that the `Read` function must be able to set the state on the id from `SetId()` only.
See https://www.terraform.io/plugin/sdkv2/resources/import#importer-state-function

I suggest to use the `group_id:lda_provider:cn` three part id. I'm aware that this requires a little refactoring. Please let me know if you want to continue working on this or not.